### ISSUE TITLE
Probably some conflict between the options?

### ISSUE THREAD

**Issue by vondele** [vondele/matetrack#105]
```
$ python ../../matetrack/matecheck.py --engine ./stockfish.master --nodes 100000 --syzygyPath /home/vondele/chess/syzygy/3-4-5-6/WDL:/home/vondele/chess/syzygy/3-4-5-6/DTZ --showAllIssues --bench --showAllStats --epdFile shortmates.epd 
Loaded 100 FENs, with max(abs(bm)) = 4.

Matetrack started for ./stockfish.master on shortmates.epd with --nodes 100000 --syzygyPath /home/vondele/chess/syzygy/3-4-5-6/WDL:/home/vondele/chess/syzygy/3-4-5-6/DTZ ...
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 100/100 [00:04<00:00, 20.37it/s]

Traceback (most recent call last):
  File "/home/vondele/chess/vondele/Stockfish/src/../../matetrack/matecheck.py", line 315, in <module>
    for _, _, pvstatus, _, _ in res:
ValueError: too many values to unpack (expected 5)
```

**Comment by vondele** [comment]
hmm, no even with basic options it fails.
```
python ../../matetrack/matecheck.py --engine ./stockfish.master --nodes 100000 --syzygyPath /home/vondele/chess/syzygy/3-4-5-6/WDL:/home/vondele/chess/syzygy/3-4-5-6/DTZ   --epdFile shortmates.epd
```

**Comment by vondele** [comment]
seems independent from the options, and introduced with the `--bench` option.

**Comment by vondele** [comment]
fixed by #106
